zerver/tests/test_audit_log.py

```diff
@@ -68,3 +69,12 @@ def test_change_bot_owner(self):
         self.assertEqual(RealmAuditLog.objects.filter(event_type='bot_owner_changed',
                                                       event_time__gte=now).count(), 1)
         self.assertEqual(bot_owner, bot.bot_owner)
+    def test_regenerate_api_key(self):
```

Comment:
Please leave one empty line between functions so that our pep8 linter won't complain.